enclave-runtime/Cargo.lock

```diff
@@ -15,7 +15,7 @@ dependencies = [
 [[package]]
 name = "ac-compose-macros"
 version = "0.1.0"
-source = "git+https://github.com/scs/substrate-api-client?branch=master#a4d931c85f30ba22ff1f982c47caf57fe43ebc90"
+source = "git+https://github.com/scs/substrate-api-client?branch=master#87595eb0136e0d7d939eedbbfbf98d62608cd8bf"
```

Comment:
This is a deliberate change, made by you? Updating the substrate-api-client version?

Reply:
Yes, I added some RPCs in the api-client needed for the current task.

Reply:
ah okay, thanks 👍